### ISSUE TITLE
ABROAD

### ISSUE THREAD

**Issue by esmanning** [carmls/snacs-guidelines#82]
> go teach English ABROAD
Reddit task_005
new_doc id = 2cb640c9-0c11-2d7f-8f3e-19dc8bcf165c
sent_id = 1

This was marked as a target, but we didn't find precedent for ABROAD in the guidelines or xposition, and weren't sure if it should really be a target. Not a real preposition, but perhaps similar to HOME?

**Comment by nschneid** [comment]
Yeah similar to HOME. It falls under the expansive definition from CGEL (e.g. you can *go home/abroad* which expresses motion toward a place like a *to*-PP).

**Comment by nschneid** [comment]
We can call these "closed-class directional locatives".

**Comment by esmanning** [comment]
Oh, this sentence might also be ambiguous - does that still apply if ABROAD modifies 'teach' rather than 'go'? (with HOME I think you'd have to add an 'at' for that)

**Comment by nschneid** [comment]
Then it's just locative. "Here/there" are other locatives that don't require a preposition. But most locations do. (The preposition is more optional with times, e.g. "I'll see you (on) Thursday.")
